src/exception_handler/exceptions.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module will contain the various exceptions.
+"""
+
+import logging
+import werkzeug.exceptions as e
+
+class NotFoundException(RuntimeError):
+    """Not found."""
+
+class MyDataNotFound(NotFoundException):
+    error_object = {}
+    def __init__(self, error_code, message, detail):
+        response = {
+            "error_code" : error_code,
+            "message" : message,
+            "detail" : detail
+        }
+        logging.info(f"response : {response}")
+        super().__init__(response)
+        self.set_error_object(response)
+
+    def set_error_object(self, response):
+        self.error_object = response
+    
+    def get_error_object(self):
+        return self.error_object
+        
+
+
+class BadRequestException(RuntimeError):
+    """Bad Request"""
+
+class MyBadRequest(BadRequestException):
```

Comment:
MyBad is not required, in-case we need to have one for BadRequest/DataNotFound..

name them appropriately, see if we can use BadRequest/NotFound only.

Reply:
Since removing the MyBadRequest was giving an error, I have renamed MyBadRequest and MyDataNotFound to BadRequestException and NotFoundException respectively.